gradle-plugin/it/src/test/java/org/eclipse/jkube/gradle/plugin/tests/ImagePullSecretsIT.java

```diff
@@ -0,0 +1,46 @@
+package org.eclipse.jkube.gradle.plugin.tests;
```

Comment:
license header missing, could you please add it by running `mvn -N license:format`?

Reply:
thx! added